processing/data_collection/gazette/spiders/pr_foz_do_iguacu.py

```diff
@@ -0,0 +1,71 @@
+import datetime as dt
+
+import scrapy
+from scrapy.http import FormRequest
+
+from gazette.items import Gazette
+
+
+class PrFozDoIguacuSpider(scrapy.Spider):
+    MUNICIPALITY_ID = '4108304'
+    name = 'pr_foz_do_iguacu'
+    allowed_domains = ['pmfi.pr.gov.br']
+    start_urls = ['http://www.pmfi.pr.gov.br/utilidades/diario/index.xhtml']
+    AVAILABLE_FROM = dt.date(2015, 1, 1)  # actually September/2007
+
+    def parse(self, response):
+        """
+        @url http://www.pmfi.pr.gov.br/utilidades/diario/index.xhtml
+        @returns requests 1 1
+        """
+        selector = '(//span[@class="ui-paginator-current"])[1]/text()'
+        paginator_text = response.xpath(selector).extract_first()
+        quantity_of_documents = int(paginator_text[1:-1].split()[-1]) + 1
```

Comment:
You can use a regex to extract this information instead of relying in string slices:

`quantity_of_documents = response.xpath('(//span[@class="ui-paginator-current"])[1]/text()').re_first('\([\d]+ de ([\d]+)\)')`